### DISCUSSION TITLE
Dropped of the radar

### DISCUSSION THREAD

**Discussion by fade2gray** [mark-wiemer/ahkpp#132] [General]
I believe there have been a couple of occasions where you have not received notifications of further comments in some threads, could [this ](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/48#event-4355805662)be one?

**Comment by mark-wiemer** [comment]
I saw it, just haven't gotten to all my notifications recently. Sorry about that, and thanks for the reminder :)
